### PR TITLE
Fixed SNAT private ranges IP validation bug

### PR DESCRIPTION
**Pull request by Gizachew-Eshetie**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

File: src/Network/Network.Test/ScenarioTests/AzureFirewallPolicyTests.ps1 [hunk 1]
```diff
@@ -1492,7 +1492,7 @@ function Test-AzureFirewallPolicyPrivateRangeCRUD {
     $location = "westus2"
     $vnetName = Get-ResourceName
     $privateRange2 = @("IANAPrivateRanges", "0.0.0.0/0", "66.92.0.0/16")
-    $privateRange1 = @("3.3.0.0/24", "98.0.0.0/8")
+    $privateRange1 = @("3.3.0.0/24", "98.0.0.0/8","10.227.16.0/20")
```
Comment (BethanyZhou) [inline src/Network/Network.Test/ScenarioTests/AzureFirewallPolicyTests.ps1:1495]: The value of $privateRange1 changes, these two test cases need to re-record.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
Others look good to me

File: src/Network/Network.Test/ScenarioTests/AzureFirewallPolicyTests.ps1 (hunk 1 above)
Comment (Gizachew-Eshetie) [reply src/Network/Network.Test/ScenarioTests/AzureFirewallPolicyTests.ps1:1495]: @BethanyZhou updated session records.

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**